src/process/dust/ccpr_scheme_fengsha_mod.F90

```diff
@@ -34,36 +34,79 @@ module CCPr_Scheme_Fengsha_Mod
    !!
    !! outputs emission flux in [ug m-2 s-1]
    !!
-   !! \param MetState     Meteorological Variables
-   !! \param DiagState    Diagnostic Variables
-   !! \param DustState    Dust Variables
-   !! \param RC           Success or Failure
+   !! \param nDustSpecies Number of dust species
+   !! \param DSOILTYPE Soil type (1-19)
+   !! \param SSM Surface soil moisture [fraction]
+   !! \param RDRAG Roughness length for momentum transfer [m]
+   !! \param TSKIN Skin temperature [K]
+   !! \param USTAR Friction velocity [m/s]
+   !! \param USTAR_THRESHOLD Friction velocity threshold [m/s]
+   !! \param GWETTOP Top level volumetric soil moisture [m3/m3]
+   !! \param z0 Roughness length [m]
+   !! \param CLAYFRAC Fraction of clay
+   !! \param SANDFRAC Fraction of sand
+   !! \param AIRDEN Air density [kg/m3]
+   !! \param FRLAND Land fraction
+   !! \param FRLANDICE Land ice fraction
+   !! \param FRSNOW Snow fraction
+   !! \param alpha Alpha scaling parameter
+   !! \param gamma Gamma scaling parameter
+   !! \param reff Effective radius
+   !! \param rlower Lower radius
+   !! \param rupper Upper radius
+   !! \param MoistOpt Moisture option
+   !! \param DragOpt Drag option
+   !! \param HorizFluxOpt Horizontal flux option
+   !! \param TotalEmission Total Emission Rate [ug/m2/s]
+   !! \param EmissionBin Emission Rate per Species [ug/m2/s]
+   !! \param RC Return code
    !!
    !! \ingroup catchem_dust_process
    !!!>
-   subroutine CCPr_Scheme_Fengsha(MetState, DiagState, DustState, RC)
+   subroutine CCPr_Scheme_Fengsha(nDustSpecies, DSOILTYPE, SSM, RDRAG, TSKIN, USTAR, USTAR_THRESHOLD, GWETTOP, z0, &
```

Comment:
I think it would be easier to compare to the call if this were one on each line too.